src/conversions.jl

```diff
@@ -278,6 +278,25 @@ julia> AstroTime.Epochs.tdbtt(tdb.jd1, tdb.jd2, AstroTime.Epochs.deltatr(tdb))
     date, date1
 end
 
+"""
+Computes difference TDB-TT in seconds at time JD (julian days)
+The timescale for the input JD can be either TDB or TT.
+
+The accuracy of this routine is approx 30 microseconds in interval 1900-2100 AD.
+Note that an accurate transformation betweem TDB and TT depends on the
+trajectory of the observer. For two observers fixed on the earth surface
+the quantity TDB-TT can differ by as much as about 4 microseconds.
+
+References:
+[1] https://www.cv.nrao.edu/~rfisher/Ephemerides/times.html#TDB
+[2] https://github.com/JuliaAstro/AstroTime.jl/issues/26
+
+"""
+function dtdb(JD)
```

Comment:
While you are at it, would you mind changing the name of both functions to something more self-explanatory? E.g. `difference_tt_tdb`. `dtdb` is a remainder of the ERFA port.

Reply:
The result is TDB minus TT
Do you want to name it difference_tt_tdb or difference_tdb_tt?